client/src/routes/routes-list.tsx

```diff
@@ -79,4 +79,113 @@ const routesList: RouteObject[] = [
 	},
 ]
 
-export default routesList
+export const routesListMentor: RouteObject[] = [
+	{
+		element: <Layout />,
+		children: [
+			{
+				path: _URL.HOME,
+				element: (
+					<ProtectedRoute>
+						<Navigate to="/meetings" />
+					</ProtectedRoute>
+				),
+			},
+			{
+				path: _URL.LOGIN,
+				element: <Login />,
+			},
+			{
+				path: _URL.LOGOUT,
+				element: <Logout />,
+			},
+			{
+				path: _URL.MEETINGS,
+				element: (
+					<ProtectedRoute>
+						<MeetingsProvider>
+							<Meetings />
+						</MeetingsProvider>
+					</ProtectedRoute>
+				),
+			},
+			{
+				path: _URL.HOMEWORKS,
+				element: (
+					<ProtectedRoute>
+						<HomeworksProvider>
+							<Homeworks />
```

Comment:
this one seems like working wrong, too
it still searches for homeworks by the name of the student. but you need to do an extra check. 

so all the process of getting homewroks should look. approximatelly like this on the backend:
- get all `homerworks` rows
- get all `students` rows
- filter all students that have this mentor assigned to them
- filter `homeworks` that contain the names of this mentor's students

Reply:
UPD: `mentor` field will be added to the HW data so you can filter it by this field

Reply:
fixed in terms of code, but needs testing after the tables are changed